src/Types.ts

```diff
@@ -48,21 +48,27 @@ export type ScopeType =
   | "arrowFunction"
   | "class"
   | "className"
+  | "collectionItem"
+  | "collectionKey"
   | "comment"
   | "dictionary"
+  | "xmlElement"
```

Comment:
Presumably this applies to react elements as well. Is there a more generic term?

Reply:
Yes because jsx is like html a super set of xml so I think the name is fitting.